public/app/plugins/datasource/elasticsearch/components/QueryEditor/BucketAggregationsEditor/SettingsEditor/DateHistogramSettingsEditor.test.tsx

```diff
@@ -0,0 +1,92 @@
+import { getDefaultTimeRange } from '@grafana/data';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { ElasticDatasource } from 'app/plugins/datasource/elasticsearch/datasource';
+import { ElasticsearchQuery } from 'app/plugins/datasource/elasticsearch/types';
+import React, { ComponentProps, ReactNode } from 'react';
+import { ElasticsearchProvider } from '../../ElasticsearchQueryContext';
+import { DateHistogram } from '../aggregations';
+import { DateHistogramSettingsEditor } from './DateHistogramSettingsEditor';
+
+const renderWithESProvider = (
```

Comment:
i want to export this in the future and use it whenever i use ElasticsearchProvider in tests.